### PR TITLE
This is what i have done to my inventory so i can still use the esx_policejob menu

### PR DESCRIPTION
**Pull request by AgenChicken24**
I gave the code in the readme so you can copy it and also I have fixed if for some reason the attachments are duplicating it's not going to happen anymore it's in the server update ammunition event. Hope you like it. Have a nice day!

### CODE REVIEW

File: ESX/Inventory HUD/README.md [hunk 1]
```diff
@@ -59,6 +59,126 @@ TriggerServerEvent('dp_inventory:updateAmmoCount', GetHashKey(weaponModel), wepI
...
+This is clined line 1092
+```lua
+local hasWeapon = false
+for number,items in pairs(PlayerData.inventory) do
+	if items.name == v.weapon then 
+		hasWeapon = true
+	end
+end
+```
+esx_policejob/client/main.lua line 1184 replace all with the code
```
Comment (iSentrie) [inline ESX/Inventory HUD/README.md:73]: You pointing a wrong lines. I suppose you have edited or different version of esx_policejob.

FOR DUMMIES.

This is what you have to replace, that whole function.
https://github.com/esx-framework/esx_policejob/blob/9ceb66a591bca41ab10786bc795ebf04e52e2877/client/main.lua#L814

File: ESX/Inventory HUD/README.md [hunk 2]
```diff
@@ -59,6 +59,126 @@ TriggerServerEvent('dp_inventory:updateAmmoCount', GetHashKey(weaponModel), wepI
 local addToSlote = true
 TriggerEvent('dp_inventory:addCurrentWeapon', weaponModel, wepInfo, addToSlote)
 ```
+
+For police job you ca do like that and when they buy it its going to have a unique weapon id.
+This is clined line 1092
```
Comment (XvenDeR) [inline ESX/Inventory HUD/README.md:64]: This line is for dragging
https://github.com/esx-framework/esx_policejob/blob/9ceb66a591bca41ab10786bc795ebf04e52e2877/client/main.lua#L1092

File: ESX/Inventory HUD/README.md (hunk 1 above)
Comment (AgenChicken24) [reply ESX/Inventory HUD/README.md:73]: yeah I guess you are right this is the server
image
this is the function that you need to change with mine in the read me
image
and this is when the menu is opened you so it does not check if you have the weapon but the item 
image

Comment (nhokhuy197) [reply ESX/Inventory HUD/README.md:73]: hey done all what you said and this is what i receive, something wrong at
local hasWeapon = false
for number,items in pairs(PlayerData.inventory) do
	if items.name == v.weapon then 
		hasWeapon = true
	end
end

Comment (AgenChicken24) [reply ESX/Inventory HUD/README.md:73]: Okay then do this replace the PlayerData.inventory in the for loop with ESX.GetPlayerData().inventory so every time you open the menu it makes sure that the inventory is updated

**Review by RuisSoftware — APPROVED** [review]

**Pull request closed without merging**